app/views/layouts/mailer.html.erb

```diff
@@ -1,13 +1,58 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <title>Barong Mailer</title>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+
     <style>
-      /* Email styles need to be inline */
+      body {
+        width: 100%;
+        height: 100%;
+        background: #F9F9F9;
+      }
+      body .body-wrap {
+        width: 100%;
+      }
+      body .body-wrap .title-header {
+        display: block;
+        text-align: center;
+      }
+      body .body-wrap .title-header img {
+        width: 200px;
+        margin: 20px 0;
+      }
+      body .body-wrap .container {
+        display: block;
+        max-width: 600px;
+        margin: 0 auto;
+        background: #FFFFFF;
+        border: 2px solid #F1F1F1;
+      }
+      body .body-wrap .container .content {
+        padding: 50px;
+        max-width: 600px;
+        margin: 0 auto;
+        display: block;
+      }
     </style>
   </head>
 
   <body>
-    <%= yield %>
+    <table class="body-wrap">
```

Comment:
are you sure table is the best way to acheive this template ?
I was thinking just a few div.

Reply:
@mod It's best practice to working with tables when we building email templates, allows us to center elements and etc. You can find a lot of interesting information on this topic, for example:
https://litmus.com/blog/the-tyranny-of-tables-why-web-and-email-design-are-so-different
https://mailbakery.com/blog/how-email-html-coding-differs-from-web-html-coding/

Reply:
can we pass parameter in this way to devise emails ?
http://guides.rubyonrails.org/action_mailer_basics.html#sending-emails-with-dynamic-delivery-options

Reply:
@amir-budaychiev we need to pass the current hostname when the email sending was originated I doubt you understand how websites table works.

we match domain name please read our code regarding custom logo in layout.